python/requirements.txt

```diff
@@ -85,6 +85,8 @@ Pillow; platform_system != "Windows"
 proxy.py
 pyarrow >= 6.0.1, < 7.0.0
 pygments
+pymongo
+pymongoarrow
 pyspark==3.1.2
```

Comment:
should `bson` be added here as well?

Reply:
It turns out pymongo has its own bson and it doesn't need to (and shouldn't)be installed.